### ISSUE TITLE
`keybase login` fails with `▶ ERROR Message wasn't signed`

### ISSUE THREAD

**Issue by Bengt** [keybase/client#5378]
I cannot login using the keybase client under Arch installed yaourt.

My log id is: 74bc3f3663eca6bd568fc31c. What I am seeing when I use my email address is:

```bash
$ keybase login
Your keybase username or email address: bengt.lueers@gmail.com
▶ WARNING Error running Bug 3964 repairman: lksec load: no device id set, thus can't fetch server half
▶ WARNING In Bug 3964 repair: lksec load: no device id set, thus can't fetch server half
▶ ERROR Message wasn't signed
$
```

When I use my keybase user name, I get a similar error (log ID b83799472162795c6aed1c1c):

```
$ keybase login
Your keybase username or email address: bengt
▶ ERROR Message wasn't signed
$
```

I can decrypt, sign and encrypt messages using `gpg` (version 2) and using keybase.io.

**Comment by Bengt** [comment]
Is #3064 related possibly?

**Comment by maxtaco** [comment]
Are you using an `EdDSA` public key?  My guess is that's the issue. We haven't rolled out support yet in the client (though the server does have support).

**Comment by Bengt** [comment]
Yes, that is the case. My key uses ECDSA and ECDH subkeys with a key length of 256, each.

**Comment by maxtaco** [comment]
hmmmm -- ecdsa?  i thought we handled ecdsa.... maybe this isn't the issue....

**Comment by maxtaco** [comment]
i'll put a ticket for us to investigate it.  can repro with `keybase id bengt` even if not signed in

**Comment by Bengt** [comment]
Yes, I get the same error message there, too:

```
$ keybase id bengt
▶ ERROR Message wasn't signed
$ 
```

**Comment by maxtaco** [comment]
OK, will hopefully get to it soon, but no guarantees :(

sorry :(

**Comment by Bengt** [comment]
No problem, thanks for the quick reply and diagnosis. I hope others will not have this issue in the future.
